.github/workflows/full-site-editing-plugin.yml

```diff
@@ -1,8 +1,9 @@
 on:
   pull_request:
-    # only trigger this workflow if FSE plugin files have been modified
+    # only trigger this workflow if FSE plugin files have been modified, or if packages have been updated.
     paths:
-    - 'apps/full-site-editing/full-site-editing-plugin/**'
+    - 'apps/full-site-editing/**'
+    - 'yarn.lock'
```

Comment:
It turns out that this match is too broad and it will trigger on PRs that are not related to FSE plugin. For example see D45192-code.

Reply:
I'll file a PR to remove it :+1:

Reply:
#43480